.infrastructure/hpa.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: autoscaling/v2
```

Comment:
The API version `autoscaling/v2` is incorrect for using resource metrics like CPU and memory utilization. Use `autoscaling/v2beta2` instead.